bokehjs/src/lib/models/annotations/color_bar.ts

```diff
@@ -393,6 +480,19 @@ export class ColorBarView extends AnnotationView {
     return max([this.model.major_tick_out, this.model.minor_tick_out])
   }
 
+  _get_extend_colorbar(): number {
```

Comment:
This function does not appear to be used anywhere outside the tests, and seems unnecessary in any case—anywhere that would use this can just compare enum values of `this.model.extend`.  This (and the corresponding tests) should be removed.